website/source/docs/guides/acl-replication.md

```diff
@@ -0,0 +1,244 @@
+---
+name: 'ACL Replication for Multiple Datacenters '
+content_length: 15
+id: acl-replication
+layout: content_layout
+products_used:
+  - Consul
+description: You can configure tokens, policies, and roles to work across multiple datacenters.
+---
+
+You can configure tokens, policies, and roles to work across multiple
+datacenters. This provides central configuration of security policies in the
```

Comment:
Do we want to use the loaded phrase "central configuration" here given the terms used to describe the L7 stuff?

Reply:
Also not sure if "security policies" is the right term to use given that we are already talking about ACL Policies in the preceding sentence.

Reply:
What about "Replicating ACL components across multiple datacenters enables failover if one of them becomes unavailable."